modules/server-node/prisma-sqlite/schema.prisma

```diff
@@ -208,19 +239,15 @@ model OnchainTransaction {
   logs              String?
   cumulativeGasUsed String?
   byzantium         Boolean?
-
-  // Channel fields
-  status         String  @default("submitted") // no enums
-  reason         String // no enums
-  error          String?
-  channelAddress String
-  channel        Channel @relation(fields: [channelAddress], references: [channelAddress])
+  status            Int?
 
   createdAt DateTime @default(now())
 
-  @@unique([transactionHash, chainId])
-  @@map(name: "onchain_transaction")
-  Transfer Transfer[] @relation("Transaction")
+  transfer Transfer? @relation
```

Comment:
This is going to create a really loopy relationship.

Imho, this should go from parent -> child -> grandchild

Transfer 1 -> 1 OnchainTransaction 1 -> 2 OnchainTransactionAttempt[]
Transfer 1 -> 1 OnchainTransaction 1 -> 2 OnchainTransactionReceipt[]